scripts/install_mujoco.py

```diff
@@ -57,7 +57,7 @@ def install_mujoco(platform, version, mujoco_path):
             "--directory-prefix",
             mujoco_path,
             mujoco_zip_url])
-    elif subprocess.call(["command", "-v", "curl"]) == 0:
+    elif subprocess.call(["which", "curl"]) == 0:
```

Comment:
```suggestion
    elif subprocess.call(["command", "-v", "curl"]) == 0:
```

Reply:
Oh oops, this should've had the `shell=True` included in it. I pushed a commit with you as a co-author. Hope that's fine.